telemetry/main.schema.json

```diff
@@ -561,10 +561,30 @@
               "required": [
                 "scalars"
               ]
+            },
+            "content": {
+              "type": "object",
+              "properties": {
+                "histograms": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "$ref": "#/definitions/histogram"
+                  }
+                },
+                "keyedHistograms": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "additionalProperties": {
+                      "$ref": "#/definitions/histogram"
+                    }
+                  }
+                }
+              }
             }
           },
           "required": [
-            "parent"
+            "parent",
+            "content"
```

Comment:
Will all pings that have the `processes` section in the payload have both of these keys?